brightchain-lib/src/lib/blocks/rawData.ts

```
@@ -13,8 +14,16 @@ import { BaseBlock } from './base';
  * It provides basic data storage and validation capabilities.
  */
 export class RawDataBlock extends BaseBlock implements IDataBlock {
+  protected static override checksumService: ChecksumService;
   private readonly _data: Buffer;
 
+  protected static override initialize() {
+    super.initialize();
```

Biome (1.9.4)

[error] Using super in a static context can be confusing.

super refers to a parent class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)